### PR TITLE
Update zram install process for upstream changes

### LINKED ISSUE #1100: reorder ztab, adjust FIND install
We need `RequireMountsFor=/opt/zram/zramX/upper` in zramsync.service

Reorder ztab so #1 is /var/log, #2 is /var/lib/openhab2/persistence, and #3 is /opt/find3/server/main
Adjust find3_setup() to reflect and add another line RequireMountsFor=/opt/zram/zram3/upper if FIND3 is installed

See also https://github.com/mstormi/openhabian-zram/pull/5#discussion_r477330296

### PR DESCRIPTION
**Pull request by ecdye**
Fixes #1100

Signed-off-by: Ethan Dye <mrtops03@gmail.com>

### CODE REVIEW

File: functions/zram.bash [hunk 1]
```diff
@@ -86,18 +86,20 @@ init_zram_mounts() {
...
-    if ! cond_redirect systemctl -q daemon-reload &> /dev/null; then echo "FAILED (daemon-reload)"; return 1; fi    
+    if ! cond_redirect systemctl disable --now zram-config.service zramsync.service; then echo "FAILED (disable service)"; return 1; fi
+    if ! cond_redirect rm -f /etc/systemd/system/zram-config.service /etc/systemd/system/zramsync.service; then echo "FAILED (remove service)"; return 1; fi
+    if cond_redirect systemctl -q daemon-reload &> /dev/null; then echo "OK"; else echo "FAILED (daemon-reload)"; return 1; fi
 
     echo -n "$(timestamp) [openHABian] Removing ZRAM... "
     if ! cond_redirect rm -f /usr/local/sbin/zram-config; then echo "FAILED (zram-config)"; return 1; fi
     if ! cond_redirect rm -f /usr/local/sbin/zramsync; then echo "FAILED (zramsync)"; return 1; fi
     if ! cond_redirect rm -f /etc/ztab; then echo "FAILED (ztab)"; return 1; fi
+    if ! cond_redirect rm -rf /storage/zram; then echo "FAILED (zramsync temp)"; return 1; fi
```
Comment (mstormi) [inline functions/zram.bash:102]: will /storage/zram be created on zramsync ? or on zram installation ? (just asking for sake of safety)

**Review by mstormi — APPROVED** [review]
LGTM.
But I'm not sure I understand the title of this PR.
You mean upstream == openhabian-zram ?

**Review by mstormi — COMMENTED** [review]
Sorry I overlooked it's draft only.

Can you patch this into the zramtest branch ? Am just about to test install another RPi.

**Comment by ecdye** [comment]
> LGTM.
> But I'm not sure I understand the title of this PR.
> You mean upstream == openhabian-zram ?

Yes

> Sorry I overlooked it's draft only.
>
> Can you patch this into the zramtest branch ? Am just about to test install another RPi.

It's already in there except one thing which I'll add right now. I actually used `zramtest` to create this. I have to wait to merge this until the PR in openhabian-zram is merged.

**Comment by mstormi** [comment]
[This one ](https://github.com/mstormi/openhabian-zram/pull/5) ?

is it done, no more open questions ?
Of course I could merge it right away but it would go live then immediately (no stable branch) so we should be sure it'll work.

**Comment by ecdye** [comment]
> [This one ](https://github.com/mstormi/openhabian-zram/pull/5) ?

Yes

> is it done, no more open questions ?

yes

> Of course I could merge it right away but it would go live then immediately (no stable branch) so we should be sure it'll work.

Sort of, most people won't be affect unless they reinstall, however once merged we need to have a patch day so we don't have any reinstall issues. I'll be more comfortable if you make a test install one more time and make sure it works for you.

**Comment by mstormi** [comment]
ok, will test first and merge eventually.
I noticed you reenabled Before=smb openhab2 which you said you wanna test that zram always comes up on boot
Have there been test, any issues?

**Comment by ecdye** [comment]
> ok, will test first and merge eventually.
> I noticed you reenabled Before=smb openhab2 which you said you wanna test that zram always comes up on boot
> Have there been test, any issues?

Yep, it seems to be working fine.

**Comment by mstormi** [comment]
ZRAM didn't start for me. Continuing in https://github.com/mstormi/openhabian-zram/pull/5

File: functions/zram.bash (hunk 1 above)
Comment (ecdye) [reply functions/zram.bash:102]: on zramsync

**Pull request merged**